proxy/test/utils.ts

```diff
@@ -0,0 +1,76 @@
+import type http from 'node:http'
+import { after, before } from 'node:test'
+
+interface TestServer {
+  address: string
+  baseUrl: string
+  port: number
+  server: http.Server
+}
+
+const __testServers: {
+  [key: string]: TestServer
+} = {}
+
+/**
```

Comment:
We do not need JSDoc here. It is better to add it everywhere or not use it.

Reply:
Just checked out other files and added a small commentary on the top of module instead of docstrings